src/components/Pagination.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="pagination">
+    <paginate
+      :page-count="pageCount"
+      containerClass="pagination__list"
+      :clickHandler="pageClicked"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import Paginate from 'vuejs-paginate';
+import { Vue, Component } from 'vue-property-decorator';
+import { Mutation, State } from 'vuex-class';
+import { DataSet } from '@/lib/dataset';
+import { numberOfPages } from '@/lib/dataset/pagination';
+import { MutationCallbacks } from '@/store/mutations';
+
+@Component({
+  name: 'pagination',
+  components: {
+    Paginate,
+  },
+})
+export default class Pagination extends Vue {
+  @State dataset!: DataSet;
+
+  @Mutation setCurrentPage!: MutationCallbacks['setCurrentPage'];
+
+  get pageCount() {
+    if (this.dataset.paginationContext) {
+      return numberOfPages(this.dataset.paginationContext);
+    }
+    return 1;
+  }
+
+  pageClicked(pageno: number) {
+    this.setCurrentPage({ pageno });
+  }
+}
+</script>
+<style lang="scss">
+.pagination {
+  text-align: center;
+  font-size: 10px;
+}
+
+.pagination__list {
+  display: inline-block;
```

Comment:
Here you should use `display flex` property and `justify-content: center` instead of display: inline-block that will horizontal align children element (see below my other comments)